front/app/containers/Admin/pagesAndMenu/containers/GenericHeroBannerForm/messages.ts

```diff
@@ -120,6 +120,20 @@ export default defineMessages({
     id: 'app.containers.AdminPage.SettingsPage.fullWidthBannerLayout',
     defaultMessage: 'Full-width banner',
   },
+  fullWidthBannerTooltip: {
+    id: 'app.containers.AdminPage.SettingsPage.fullWidthBannerTooltip',
+    defaultMessage:
+      'This banner stretches over the full width for a great visual effect. The image will try to cover as much space as possible, causing it to not always be visible at all times. You can combine this banner with an overlay of any colour. More info on the recommended image usage can be found on our { link }.',
+  },
+  fullWidthBannerTooltipLink: {
+    id: 'app.containers.AdminPage.SettingsPage.fullWidthBannerTooltipLink',
+    defaultMessage: 'knowledge base',
+  },
+  fullWidthBannerTooltipLinkUrl: {
+    id: 'app.containers.AdminPage.SettingsPage.fullWidthBannerTooltipLinkUrl',
+    defaultMessage:
+      'https://support.citizenlab.co/en/articles/1346397-what-are-the-recommended-dimensions-and-sizes-of-the-platform-images',
```

Comment:
Do we have any plans to use different URLs? If not, I would keep it simple and use just one URL (if we need to update this URL, we'll do it just in one place). But I believe it's just a personal preference, so up to you.

Reply:
Not sure - but I think you are right, probably a good idea to use one url here. Will change 👍